__tests__/app.test.js

```diff
@@ -18,6 +18,18 @@ afterAll(() => {
   return disconnect();
 });
 
+describe("404s on non-existent endpoints", () => {
+  test("status:404, responds with an appropriate error message", () => {
```

Comment:
more tests for diifferent errors?

Reply:
Error tests will be described in each endpoint, the only standalone test we need is 404 for now